File: BOJ/CLASS 2/2751.js
// 수 정렬하기 2

const input = require('fs')
  .readFileSync('예제.txt')
  .toString()
  .trim()
  .split('\n')
  .map((x) => Number(x));

function solution(input) {
  const [N, ...arr] = input;

  arr.sort((a, b) => a - b);

  return arr.join('\n');
}

console.log(solution(input));

File: BOJ/CLASS 2/Python/2751.py
# 수 정렬하기 2
import sys

input = sys.stdin.readline
n = int(input())
nums = [int(input()) for _ in range(n)]

# sort(): Timsort 알고리즘을 사용하여 O(nlogn)의 시간복잡도를 보장
nums.sort()

print('\n'.join(map(str, nums)))